contents/tutorials/delayed-survey.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How to show a survey after a delay
+date: 2023-11-01
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys']
+---
+
+Before asking users about your app, you want them to **actually use it**. Surveys that pop up right away hinder their initial experience and get less useful responses as a result.
+
+To help you do prevent this, you can set up a delay surveys and this tutorial shows you how. It goes over how to create a survey targeting a HTML class in PostHog and then implement a delay using that class and survey in your app.
+
+## Create your delayed survey
+
+First, we create our class-targeted survey in PostHog by going to the [Surveys tab](https://app.posthog.com/surveys) and clicking "New survey." Choose any template or create a blank survey, fill out the details, and open the "Targeting" section. 
+
+In the targeting section, we need to set up the class targeting. Start by switching from "All users" to "Users who match…". Add `.delayed-survey` under "Selector matches" (the **leading period** is important) and click "Save as draft." Our implementation will add this class to the page when want the survey to show.
+
+Finally, click Launch. You don’t have to worry about this survey showing early, because the class selector won’t match anything until we implement it.
+
+![Creating a survey video](../images/tutorials/delayed-survey/survey.mp4)
+
+## Implement your delayed survey
+
+Once installed in your app using the SDK or snippet, PostHog automatically shows your survey to targeted users. All we need to do is set up the logic to do the delay and add the class to our page.
```

Comment:
Not sure why we mention SDK or snippet here all of sudden? We didnt mention it previously